lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -8,57 +8,80 @@
 @import '../../scss-shared/nx-text-helpers';
 @import '../../scss-shared/nx-container-helpers';
 
+$nx-dropdown-width: 250px;
+
 .nx-dropdown {
   display: inline-block;
   position: relative;
+  width: $nx-dropdown-width;
 }
 
 .nx-dropdown__toggle {
+  display: flex;
   margin: 0;
   outline: none;
+  text-align: left;
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+  width: $nx-dropdown-width;
+
+  span {
```

Comment:
Considering that the label content can be arbitrary JSX, I worry that this selector isn't specific enough

Reply:
> Considering that the label content can be arbitrary JSX, I worry that this selector isn't specific enough

The `<span>` comes from `import { ensureElement } from '../../util/reactUtil';`, if I wrap `{ labelContent }` with a `<span>` I get a `<span>` within a `<span>`. SO I don't know how to put a class on that `<span>`

Reply:
> > Considering that the label content can be arbitrary JSX, I worry that this selector isn't specific enough
> 
> The `<span>` comes from `import { ensureElement } from '../../util/reactUtil';`, if I wrap `{ labelContent }` with a `<span>` I get a `<span>` within a `<span>`. SO I don't know how to put a class on that `<span>`

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/6eb99a4334e0c4ea78aef1d7282b956f2ca1cb4c